Themes/catppuccin-frappe.theme.json

```
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Frappé",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/the-archive"
+    }
+  ],
```

Maybe add another JSON object here to link to the original Catppuccin repo or mention the original author(s) instead? As far as I'm concerned, your name on top as the author of this file, then something to acknowledge the community behind it?

```suggestion
  "author": [
    {
      "name": "Roger Steve Ruiz",
      "url": "https://rog.gr/"
    },
    {
      "name": "Catppuccin Organization",
      "url": "https://github.com/catppuccin/the-archive"
    }
  ],
```

Something like that would be parseable and visible in the app; the "basedOn" key would not be a.t.m. and just be flavor for readers of this code :)